projects/admin/src/app/acquisition/components/account/account-detail-view/account-detail-view.component.scss

```diff
@@ -0,0 +1,21 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2021 RERO
+ * Copyright (C) 2021 UCLouvain
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+.amount {
+  font-family: Arial;
```

Comment:
Arial, seriously? :smile: 

If it is for numbers, why not just `font-family: 'Courier New', Courier, monospace;`?

Reply:
I tried, but it's not pretty with current font. 
Arial is also a monospace font, installed on al OS.